src/scripts/core/visualization-container.js

```diff
@@ -74,6 +74,12 @@
             return this.ctx;
         },
 
+        setVisibility: function (visible) {
+            var node = this.layer.node();
+
+            visible ? $(node).show() : $(node).hide();
```

Comment:
jquery is a contour dependency. regardless, I could use pure javascript (node.css('display', 'none') or even possibly d3. IMHO, doesn't really matter, as long as it works, which the above jsfiddle proves.